stubs/xoops_database.stub.php

```
@@ -18,6 +18,12 @@ public function query($sql) {}
      */
     public function queryF($sql) {}
 
+    /**
+     * @param string $sql
+     * @return \mysqli_result|bool
+     */
+    public function exec($sql) {}
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$sql'. (undefined)

(UnusedFormalParameter)